src/promptflow-rag/promptflow/rag/_build_mlindex.py

```diff
@@ -72,15 +72,24 @@ def build_index(
         )
         raise e
 
-    if not embeddings_model_config.embeddings_model:
-        raise ValueError("Please specify embeddings_model_config.embeddings_model")
-    embeddings_model = build_open_ai_protocol(embeddings_model_config.embeddings_model)
+    if not embeddings_model_config.model_name:
+        raise ValueError("Please specify embeddings_model_config.model_name")
+
+    if "cohere" in embeddings_model_config.model_name:
```

Comment:
Is the plan ultimately to have a "supported_models" list that we can easily add new models to?

Reply:
yes, this should be addressed as a follow up task